packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -655,6 +655,21 @@
             "name": "aws_hideRegion",
             "description": "Records a call to remove a region from the explorer"
         },
+        {
+            "name": "lambda_createServerlessLandProject",
```

Comment:
It looks like this change can be dropped since it is now in common: https://github.com/aws/aws-toolkit-common/pull/996/files

Reply:
Done